src/main/kotlin/uk/gov/justice/hmpps/offenderevents/service/OffenderEventListener.kt

```diff
@@ -25,7 +25,7 @@ class OffenderEventListener(
     val log: Logger = LoggerFactory.getLogger(this::class.java)
   }
 
-  @SqsListener(queueNames = ["event"], factory = "hmppsQueueContainerFactoryProxy")
+  @SqsListener(queueNames = ["event"], factory = "hmppsQueueContainerFactoryProxy", maxInflightMessagesPerQueue = "1", maxMessagesPerPoll = "1")
```

Comment:
Ignore async capabilities by only allowing 1 message at a time.

Reply:
don't think I would call it `async`. more multi threaded.